src/components/cell-control/input.test.jsx

```diff
@@ -29,7 +29,15 @@ describe('Input cell control', () => {
   };
 
   it('has defaults', () => {
```

Comment:
I think in other components we usually have a `it('supports ref api')` type test

Reply:
I think it should just be part of the "default tests". I've found that we always want to test 3 things for any component:

- DOM markup
- Initial focus
- Initial ref structure